index.js

```diff
@@ -372,7 +359,7 @@ function $asStringSmall (str) {
       surrogateFound = true
     }
     if (point === 34 || point === 92) {
-      result += str.slice(last, i) + '\\'
+      result += str.slice(last, i) + '\\\\'
```

Comment:
Why this change?

Reply:
$asStringSmall is now inside an es6 template string. Inside those strings, slashes need to be escaped.